src/main/resources/openapi/transcriptions.yaml

```diff
@@ -535,6 +591,33 @@ components:
           example: 3
         workflow_comment:
           type: string
+
+    UpdateTranscriptionsRequest:
+      properties:
+        transcriptions:
+            type: array
+            items:
+              $ref: '#/components/schemas/UpdateTranscriptions'
+
+    UpdateTranscriptions:
+      type: object
+      required:
+        - transcriptionId
+      properties:
+        transcriptionId:
```

Comment:
This property should be named like others in transcriptions api - i.e. `transcription_id`
The standards prefer snake case naming convention for JSON.

Reply:
renamed